Cartfile.resolved

```diff
@@ -1,17 +1,17 @@
-github "getsentry/sentry-cocoa" "3.12.4"
-github "Alamofire/Alamofire" "4.3.0"
-github "sleroux/Deferred" "35b8927c1b94ce074e10793c57e1f80d0e2227fa"
-github "cezheng/Fuzi" "2.0.2"
-github "swisspol/GCDWebServer" "3.3.2"
-github "kif-framework/KIF" "v3.7.3"
-github "SnapKit/SnapKit" "3.1.2"
-github "mozilla-mobile/SwiftKeychainWrapper" "218a97f3882ab1098a2ad4c706c13c606446da18"
-github "DaveWoodCom/XCGLogger" "Version_4.0.0"
-github "adjust/ios_sdk" "v4.12.1"
 github "AgileBits/onepassword-app-extension" "bcc4cc97fed9a6e73fa204f2e61138e353cb3ef7"
+github "Alamofire/Alamofire" "4.7.2"
+github "DaveWoodCom/XCGLogger" "Version_4.0.0"
+github "SnapKit/SnapKit" "3.1.2"
 github "SwiftyJSON/SwiftyJSON" "3.1.4"
+github "adjust/ios_sdk" "v4.12.1"
+github "cezheng/Fuzi" "2.0.2"
 github "farhanpatel/JSONSchema.swift" "1c052b83baa8c497e12cde6a8afca0f54574612f"
+github "getsentry/sentry-cocoa" "3.12.4"
 github "google/EarlGrey" "1.13.0"
-github "rs/SDWebImage" "4.1.0"
-github "mozilla-mobile/telemetry-ios" "v1.1.0"
+github "kif-framework/KIF" "v3.7.3"
 github "mozilla-mobile/MappaMundi" "0b4f70e2feb941f4d56bd9e27e408d4300656a87"
+github "mozilla-mobile/SwiftKeychainWrapper" "46e5980ee7630be015aae6a6307c2fe1e8f3661d"
+github "mozilla-mobile/telemetry-ios" "v1.1.0"
+github "rs/SDWebImage" "4.1.0"
+github "sleroux/Deferred" "35b8927c1b94ce074e10793c57e1f80d0e2227fa"
+github "swisspol/GCDWebServer" "3.3.2"
```

Comment:
When you run carthage it might update this file but we should make sure that the changes are not commited in git. unless we are consciously updating the dependencies. Could you please remove changes to this file?